data/huion-kamvas-24-gs2401.tablet

```diff
@@ -1,18 +1,17 @@
-# This tablet description file has been generated using an automated tool:
-# https://github.com/JoseExposito/uclogic-libwacom-tablet-descriptions
-# If you own this tablet, please improve it.
-#
-# sysinfo missing - if you own this device please provide it. See this link
-# for details: https://github.com/linuxwacom/libwacom/wiki/Adding-a-new-device
+# Huion
+# Kamvas 24
+# GS2401
+
+# sysinfo.gc7uyE9JXK
+# https://github.com/linuxwacom/wacom-hid-descriptors/issues/398
 
 [Device]
 Name=KAMVAS 24 GS2401
 ModelName=KAMVAS 24 GS2401
 Class=Cintiq
-# Product ID is unknown, please fix
-DeviceMatch=usb|256c|006e||HUION_M206;usb|256c|006d||HUION_M206;usb|256c|006f||HUION_M206;usb|256c|0064||HUION_M206;
-Width=9  # autogenerated value, please fix
-Height=6  # autogenerated value, please fix
+DeviceMatch=usb|256c|006d;
```

Comment:
This we can't do - you're now matching on every single device with the 006d PID and potentially (well, definitely) breaking all other devices with the same PID. This needs to have the firmware specifier in it so it only matches on the kamvas 24.
```
DeviceMatch=usb|256c|006d||HUION_M206
```
alternatively/in addition if the device has a unique device name we can use that here too, see e.g. `data/huion-h640p.tablet`

Reply:
So shoud it be like this ?
`DeviceMatch=usb|256c|006d|HUION Huion Tablet_GS2401 Tablet Monitor Pen;` ?
I assume we don't need Pad because there are no buttons on Kamvas 24 display tablet.
Kamvas 24 has built in monitor, looking at huion-kamvas-13.tablet file should this be also included ? 
`usb|256c|006d|Tablet Monitor;`

Reply:
```
 DeviceMatch=usb|256c|0006d||HUION_M206;usb|256c|006d|HUION Huion Tablet_GS2401 Tablet Monitor Pen;
```
This way we have both the firmware match and the name match (the latter may prove to be less reliable although this one at least has the model number in the name).

I doubt the `tablet monitor` device doesn't need an entry - maybe the `Tablet Monitor Pen` if it sends events? Looking at the kamvas 13 file I'm not sure this is really needed tbt.